tracing-opentelemetry/Cargo.toml

```diff
@@ -31,11 +31,15 @@ tracing-core = { path = "../tracing-core", version = "0.1.28" }
 tracing-subscriber = { path = "../tracing-subscriber", version = "0.3.0", default-features = false, features = ["registry", "std"] }
 tracing-log = { path = "../tracing-log", version = "0.1.3", default-features = false, optional = true }
 once_cell = "1.13.0"
+instant = "0.1.12"
```

Comment:
would it make sense to move this dependency to `[target.wasm32-unknown-unknown.dependencies]` as well? this isn't a big deal, but it would be nice to avoid downloading and compiling a crate that isn't needed on builds for non-wasm targets.

Reply:
I'm unhappy with some of `instant`'s design decisions myself, but AFAIK it's currently the best/most widely used available implementation of a portable `Instant` type. This also means that it's somewhat likely to already be somewhere in a downstream crate's dependency graph anyways. It feels redundant to replicate here what's already implemented over there: a simple switching mechanism that just reexports `std` on non-wasm targets